<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>349</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>349</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NetTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lblUdpIp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUdpSend">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTcpSend">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lblTcpIp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lblUdpPort">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>14253</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lblTcpPort">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>13254</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Multicast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMulticastSend">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lblMulticastIp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>239.9.9.9</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lblMulticastPort">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>39999</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tboxOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>331</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cboxTcpConnection">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>굴림</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScrolltoBottom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scroll to Bottom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectProtocol">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>300</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Protocol</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lblUdpIp</tabstop>
  <tabstop>lblUdpPort</tabstop>
  <tabstop>btnUdpSend</tabstop>
  <tabstop>cboxTcpConnection</tabstop>
  <tabstop>lblTcpIp</tabstop>
  <tabstop>lblTcpPort</tabstop>
  <tabstop>btnTcpSend</tabstop>
  <tabstop>lblMulticastIp</tabstop>
  <tabstop>lblMulticastPort</tabstop>
  <tabstop>btnMulticastSend</tabstop>
  <tabstop>tboxOutput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
